File: dotnet/weatherapp-route-to-code/WeatherForecast.cs
using System;
using System.Text.Json.Serialization;

namespace weatherapp_route_to_code
{
    public class WeatherForecast
    {
        [JsonPropertyName("location")]
        public string Location { get; set; }

        [JsonPropertyName("forecast")]
        public string Forecast { get; set; }
    }
}

File: java/weatherapp-springboot/src/main/java/com/rynowak/weatherapp/WeatherForecast.java
package com.rynowak.weatherapp;

public class WeatherForecast {
    private String location;
    private String forecast;

    public WeatherForecast(String location, String forecast) {
        this.location = location;
        this.forecast = forecast;
    }

    public String getLocation() {
        return location;
    }

    public String getForecast() {
        return forecast;
    }
}